### PR TITLE
[ROSTESTS] Disable some tests on x64 for now

### PR DESCRIPTION
**Pull request by tkreuzer**
Try to make Test WHS x64 happy
See CORE-17664

### CODE REVIEW

**Review by binarymaster — APPROVED** [review]
Tester looks happy and so me. :smile: 

image

File: modules/rostests/apitests/localspl/tests.c [hunk 1]
```diff
@@ -210,6 +210,14 @@ _RunRemoteTest(const char* szTestName)
 
 START_TEST(fpEnumPrinters)
 {
+#if defined(_M_AMD64)
+    if (!winetest_interactive)
+    {
+        skip("ROSTESTS-366: Skipping localspl_apitest:fpEnumPrinters because it hangs on Windows Server 2003 x64-Testbot. Set winetest_interactive to run it anyway.\n");
```
Comment (ColinFinck) [inline modules/rostests/apitests/localspl/tests.c:216]: All localspl API tests hang on the Win2003 x64 Testbot the same way. It's just that you won't notice now as we're running tests under a restricted user account.

File: modules/rostests/kmtests/ntos_ex/ExFastMutex.c [hunk 2]
```diff
@@ -310,6 +310,14 @@ TestFastMutexConcurrent(
 
 START_TEST(ExFastMutex)
 {
+#if defined(_M_AMD64)
+    if (TRUE)
+    {
+        skip(FALSE, "ROSTESTS-367: Skipping kmtest:ExFastMutex because it hangs on Windows Server 2003 x64-Testbot. Set winetest_interactive to run it anyway.\n");
```
Comment (ColinFinck) [inline modules/rostests/kmtests/ntos_ex/ExFastMutex.c:316]: Misleading comment, `winetest_interactive` won't change anything for this kmtest

File: modules/rostests/apitests/localspl/tests.c (hunk 1 above)
Comment (tkreuzer) [reply modules/rostests/apitests/localspl/tests.c:216]: Ok, will skip all of them.

**Pull request merged**